tests/unit/orchestrator/search_planner/test_multi_tier_error_handling.py

```
@@ -0,0 +1,479 @@
+        _, var2 = pair
+        planner.tell(var2, [_success_result(var2, throughput=50.0)])
+        # At this point: fast has infeasible_min=2, economy has infeasible_min=2
+
+        # Now we're in bisect phase — but let's force a non-monotonic probe
+        # by directly manipulating and telling at a value above infeasible_min
+        # that passes. The next ask should be a midpoint.
+        pair = planner.ask()
+        if pair is None:
+            return  # planner converged early due to bracket size
+
+        _, var3 = pair
```

Ruff (0.15.15)

[warning] Unpacked variable `var3` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)